buddy-guide.md

```diff
@@ -0,0 +1,67 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+**Table of Contents**
+
+- [Buddy Guide](#buddy-guide)
+- [Before the new student starts](#before-the-new-student-starts)
+- [Day 1](#day-1)
+- [Week 1](#week-1)
+- [End of Week 1](#end-of-week-1)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Buddy Guide
+
+As part of the onboarding process, each new student will be allocated a buddy. A buddy is typically an experienced team member who will be
```

Comment:
The `onboarding process` doesn't make sense for this context. Remove this. Maybe replace the sentence with `Each student partaking in the internship programme will be allocated a buddy`?